specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -782,6 +782,20 @@
             "type": "string",
             "description": "The name of the dedicated host group."
           },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation. The response shows the list of instance view of the dedicated hosts under the dedicated host group.",
+            "enum": [
+              "instanceView"
+            ],
+            "x-ms-enum": {
+              "name": "InstanceViewTypes",
+              "modelAsString": false
```

Comment:
>"modelAsString": false [](start = 14, length = 22)

modelAsString: true is highly recommended to avoid breaking SDKs in the future. Is there a reason not to use it here?

Reply:
Setting it to true here breaks SDK generation with the error message below. I believe this is because our Swagger spec already has x-ms-enum with modelAsString set to false in a few places.

"Swagger document contains two or more x-ms-enum extensions with the same name 'InstanceViewTypes' and different values: instanceView vs. instanceView"